modules/ROOT/pages/Development/Satisfactory/Audio.adoc

```diff
@@ -5,20 +5,367 @@
+
+* Verify that the `Wwise Short ID` matches the event ID in Wwise.
+* Verify the `Wwise Name` matches the event name.
+
+image:Satisfactory/Wwise/Wwise_UEAkEvent_Verify.png[Wwise_UEAkEvent_Verify, 500]
+
+If everything has worked correctly, you can now test the event by right-clicking the event in Unreal and clicking `Play Event`.
+This will trigger the event in the editor, for example, playing the sound if the event is a Play-type event.
+
+=== Triggering AKEvents in Code
+
+Triggering AKEvents is typically done in Blueprint code.
+Although it is possible to trigger them from {cpp}, this is not recommended, as it would require passing lots of asset references to the {cpp} side. If you want to trigger events from {cpp}, define a `BlueprintImplementableEvent` UFUNCTION for your {cpp} code to call, allowing you to define the implementation in Blueprint.
+
+==== Fire & Forget Events
+
+If you would like to play a one-off AK event, such as a spark sound, you will need to use the following method in your blueprint.
+
+image:Satisfactory/Wwise/Wwise_UEAkEvent_FireAndForget.png[Wwise_UEAkEvent_FireAndForget, 100%]
+
+This will create a new temporary `AK Component` and then play the AK Event specified.
+
+Once the AK event has ended, the temporary `AK Component` will be automatically cleaned up.
+
+==== Looping Events
+
+If you would like to play a sound effect that might loop, for example, a humming/buzzing sound for a building, you should use the same AK Component for ease of reference and to avoid the creation/destruction overhead. Add an `AK Component` to your actor blueprint and set the `AK Audio Event` variable on the component.
+
+Make sure to stop the audio when the actor is destroyed,
+otherwise it will continue to play with no way to stop it.
+
+image:Satisfactory/Wwise/Wwise_UEAkEvent_EndPlay.png[Wwise_UEAkEvent_EndPlay, 100%]
+
+=== Updating Wwise Values From Your Mod
+
+Wwise uses a feature called Real-Time Parameter Control (RTPC) that allows game engines like Unreal or Unity to change values on Wwise assets in real-time.
+This enables special behaviors like a manufacturing building's sound volume to fluctuate based on recipe progress.
+
+After setting up RTPC in Wwise and Unreal, blueprint code can update audio parameter values.
+
+To set these up, in your Wwise project, go to Project Explorer and the `Game Syncs` tab.
+
+Under `Game Parameters -> Default Work Unit` create a new game parameter by right-clicking the `Default Work Unit` and then `New Child -> Game Parameter`
+
+image:Satisfactory/Wwise/Wwise_RTPC_Create.png[Wwise_RTPC_Create, 100%]
+
+Name this whatever you want.
+
+Open the newly created Game Parameter, and you should see the following in the main section:
+
+image:Satisfactory/Wwise/Wwise_RTPC_Range.png[Wwise_RTPC_Range, 100%]
+
+The parameter must have a range defined.
+A useful range is `0.0` to `1.0` since many existing gameand engine systems already use that range as a start-finish, minimum-maximum, or off-on scale.
```

Comment:
```suggestion
A useful range is `0.0` to `1.0` since many existing game and engine systems already use that range as a start-finish, minimum-maximum, or off-on scale.
```